test/cloudops.go

```diff
@@ -59,6 +64,36 @@ func compute(t *testing.T, driver cloudops.Ops) {
 	groupInfo, err := driver.InspectInstanceGroupForInstance(instanceID)
 	require.NoError(t, err, "failed to inspect instance group")
 	require.NotNil(t, groupInfo, "got nil instance group info from inspect")
```

Comment:
test with both regional and zonal clusters.

Reply:
Tested these changes with both regional (spread across multiple zone) and zonal (single zone) clusters.